src/els_completion_provider.erl

```diff
@@ -123,7 +123,7 @@ find_completion(_Prefix, _TriggerKind, _Opts) ->
   null.
 
 %%==============================================================================
-%% Modules
+%% modules
```

Comment:
Why? 🗡

Reply:
I don't know!!! :disappointed:

Reply:
I will revert this change and merge the PR.